devtools/blade_runner/subtitles/module.mk

```diff
@@ -0,0 +1,2 @@
+
+MODULE := devtools/blade_runner/subtitles
```

Comment:
It looks that the building rules are missing here. E.g. we would need to have to demonstrate how to build it.